metadata-db-app/src/migrations/050_setup_variable_associations_table.clj

```diff
@@ -0,0 +1,60 @@
+(ns migrations.050-setup-variable-associations-table
+  (:require
+   [config.mdb-migrate-helper :as h]))
+
+(def ^:private variable-assocs-column-sql
+  "id NUMBER,
+  concept_id VARCHAR(255) NOT NULL,
+  native_id VARCHAR(1500) NOT NULL,
+  metadata BLOB NOT NULL,
```

Comment:
@ygliuvt that is incorrect. Tombstones have NULL for the metadata field for all concept types. Tag associations may be different from the rest, but I doubt it.